components/automate-cli/pkg/verifyserver/services/s3configservice/s3configservice_test.go

```diff
@@ -0,0 +1,105 @@
+package s3configservice_test
```

Comment:
please write the proper unit tests

Reply:
we need to hit actual aws function to get success response, so only failure unit test case is written